src/brain/requiredChecks/handleNewFailedBuild.ts

```diff
@@ -215,7 +215,7 @@ export async function handleNewFailedBuild({
   // DEBUG: Not sure why this messages sometimes does not get threaded.
   // The Slack API should make this threaded because existingFailureMessage exists
   // eslint-disable-next-line no-console
-  console.log({ existingFailureMessage, followupFailureMessage });
+  // console.log({ existingFailureMessage, followupFailureMessage });
```

Comment:
Should we keep this around?

Reply:
Isn't it bad practice to have console logging in prod? (I'm not sure why the log was needed originally)